### PR TITLE
[IMP] toctrees: redesign and new card component

### PR DESCRIPTION
**Pull request by mano-odoo**
The goal of this commit is to propose a first step towards a new `.. card::` directive for the toctrees. This is a first design proposition for this new card component in raw html for the fallback page `howtos.rst`

The long term objective is to redesign the toctrees by implementing card components, clearer toctree list design and custom html pages.

Related to task-3059229

### CODE REVIEW

File: content/developer/howtos.rst [hunk 1]
```diff
@@ -1,8 +1,117 @@
...
+
+  <div>
+  <!-- First type of card - Full width card -->
+    <div class="row mb-5 center">
+        <div class="col-12 col-xl-8 col-xxl-6">
+          <a class="o_toctree_card" href="howtos/rdtraining.html">
+            <div class="card">
+              <div class="card-body pb-0">
+                <h3 class="card-title text-primary mb-1">Getting started</h3>
+                <p class="card-text text-dark fw-normal">Learn the most important parts of the Odoo development framework to create your own modules with  and for Odoo in this step-by-step tutorial crafted for newcomers and any other individual curious about Odoo development.</p>
```
Comment (edi-odoo) [inline content/developer/howtos.rst:17]:
```suggestion
                <p class="card-text text-dark fw-normal">Learn the most important parts of the Odoo development framework to create your own modules with and for Odoo in this step-by-step tutorial crafted for newcomers and any other individual curious about Odoo development.</p>
```

File: content/developer/howtos.rst [hunk 2]
```diff
@@ -1,8 +1,117 @@
...
+
+    <!-- NEW ROW WITH SMALL CARDS -->
+    <div class="row row-cols-1 row-cols-md-2 row-cols-xl-3 row-cols-xxl-4 g-4 mb-4">
+      <!-- 2nd type of card - Small card with badge rounded-pill -->
+      <div class="col">
+        <a class="o_toctree_card" href="howtos/company.html">
+          <div class="card h-100">
+            <div class="card-body pb-0">
+              <h4 class="card-title text-primary mb-1">Building a website</h4>
+              <p class="card-text text-dark fw-normal">Build your first website modules with Odoo</p>
```
Comment (edi-odoo) [inline content/developer/howtos.rst:37]:
```suggestion
              <p class="card-text text-dark fw-normal">Build your first website modules with Odoo.</p>
```

File: content/developer/howtos.rst [hunk 3]
```diff
@@ -1,8 +1,117 @@
...
+        </a>
+      </div>
+
+      <!-- 3rd type of card - Small card WITHOUT badge rounded-pill -->
+      <div class="col">
+        <a class="o_toctree_card" href="howtos/company.html">
+          <div class="card h-100">
+            <div class="card-body pb-0">
+              <h4 class="card-title text-primary mb-1">Building a module</h4>
+              <p class="card-text text-dark fw-normal">Learn how to build your own Odoo modules</p>
```
Comment (edi-odoo) [inline content/developer/howtos.rst:52]:
```suggestion
              <p class="card-text text-dark fw-normal">Learn how to build your own Odoo modules.</p>
```

File: content/developer/howtos.rst [hunk 4]
```diff
@@ -1,8 +1,117 @@
...
+          </div>
+        </a>
+      </div>
+
+      <div class="col">
+        <a class="o_toctree_card" href="howtos/profilecode.html">
+          <div class="card h-100 pb-0">
+            <div class="card-body">
+              <h4 class="card-title text-primary mb-1">Profiling Odoo code</h4>
+              <p class="card-text text-dark fw-normal">Learn more about Odoo's embedded profiler of code.</p>
```
Comment (edi-odoo) [inline content/developer/howtos.rst:66]:
```suggestion
              <p class="card-text text-dark fw-normal">Learn more about Odoo's embedded code profiler.</p>
```

File: content/developer/howtos.rst [hunk 5]
```diff
@@ -1,8 +1,117 @@
...
+          </div>
+        </a>
+      </div>
+
+      <div class="col">
+        <a class="o_toctree_card" href="howtos/company.html">
+          <div class="card h-100 pb-0">
+            <div class="card-body">
+              <h4 class="card-title text-primary mb-1">Multi-company Guidelines</h4>
+              <p class="card-text text-dark fw-normal">Access information from multiple companies but also to create/edit records in a multi-company environment
```
Comment (edi-odoo) [inline content/developer/howtos.rst:78]:
```suggestion
              <p class="card-text text-dark fw-normal">See how to manage multiple companies and create/edit records in a multi-company environment.
```

File: content/developer/howtos.rst [hunk 6]
```diff
@@ -1,8 +1,117 @@
...
+            </div>
+            <div class="card-footer border-0"></div>
+          </div>
+        </a>
+      </div>
+
+      <div class="col">
+        <a class="o_toctree_card" href="howtos/company.html">
```
Comment (edi-odoo) [inline content/developer/howtos.rst:86]: Wrong URL

File: content/developer/howtos.rst [hunk 7]
```diff
@@ -1,8 +1,117 @@
...
+            <div class="card-footer border-0">
+              <span class="badge rounded-pill bg-dark mt-auto mb-2">Beginner</span>
+            </div>
+          </div>
+        </a>
+      </div>
+
+      <!-- 3rd type of card - Small card WITHOUT badge rounded-pill -->
+      <div class="col">
+        <a class="o_toctree_card" href="howtos/company.html">
```
Comment (edi-odoo) [inline content/developer/howtos.rst:48]: Wrong URL

File: content/developer/howtos.rst [hunk 8]
```diff
@@ -1,8 +1,117 @@
...
+        </div>
+    </div>
+
+.. raw:: html
+
+    <!-- NEW ROW WITH SMALL CARDS -->
+    <div class="row row-cols-1 row-cols-md-2 row-cols-xl-3 row-cols-xxl-4 g-4 mb-4">
+      <!-- 2nd type of card - Small card with badge rounded-pill -->
+      <div class="col">
+        <a class="o_toctree_card" href="howtos/company.html">
```
Comment (edi-odoo) [inline content/developer/howtos.rst:33]: Wrong URL

File: content/developer/howtos.rst [hunk 9]
```diff
@@ -1,8 +1,117 @@
...
+              <h4 class="card-title text-primary mb-1">Accounting localization</h4>
+              <p class="card-text text-dark fw-normal">Build a localization module, bank operation model and create dynamic report</p>
+            </div>
+            <div class="card-footer border-0"></div>
+          </div>
+        </a>
+      </div>
+
+      <div class="col">
+        <a class="o_toctree_card" href="howtos/company.html">
```
Comment (edi-odoo) [inline content/developer/howtos.rst:98]: Wrong URL

File: content/developer/howtos.rst [hunk 10]
```diff
@@ -1,8 +1,117 @@
...
+          <div class="card h-100 pb-0">
+            <div class="card-body">
+              <h4 class="card-title text-primary mb-1">Accounting localization</h4>
+              <p class="card-text text-dark fw-normal">Build a localization module, bank operation model and create dynamic report</p>
```
Comment (edi-odoo) [inline content/developer/howtos.rst:90]:
```suggestion
              <p class="card-text text-dark fw-normal">Read how to build a localization module, create bank operation models and dynamic reports.</p>
```

File: content/developer/howtos.rst [hunk 11]
```diff
@@ -1,8 +1,117 @@
...
+          </div>
+        </a>
+      </div>
+
+      <div class="col">
+        <a class="o_toctree_card" href="howtos/company.html">
+          <div class="card h-100 pb-0">
+            <div class="card-body">
+              <h4 class="card-title text-primary mb-1">Translating Modules</h4>
+              <p class="card-text text-dark fw-normal">Implicit and explicit export of translatable term</p>
```
Comment (edi-odoo) [inline content/developer/howtos.rst:102]:
```suggestion
              <p class="card-text text-dark fw-normal">See how to provide translation abilities to your module.</p>
```

File: extensions/odoo_theme/static/style.scss [hunk 12]
```diff
@@ -1162,3 +1162,18 @@ footer {
 
     }
 }
+
+.o_toctree_card {
+    transition: all 200ms;
```
Comment (edi-odoo) [inline extensions/odoo_theme/static/style.scss:1167]:
```suggestion
```

File: extensions/odoo_theme/static/style.scss [hunk 13]
```diff
@@ -1162,3 +1162,18 @@ footer {
...
+    &:hover {
+        text-decoration: none;
+        cursor: pointer;
+
+        .card {
+            background-color: rgba($teal-100, 0.2);
+        }
+
+    }
+
```
Comment (edi-odoo) [inline extensions/odoo_theme/static/style.scss:1178]:
```suggestion

    .card {
        transition: all 200ms;
    }
```

File: content/developer/howtos.rst [hunk 14]
```diff
@@ -1,8 +1,117 @@
...
+:hide-page-toc:
 
 =========
 Tutorials
 =========
+.. raw:: html
+
+  <div>
+  <!-- First type of card - Full width card -->
+    <div class="row mb-5 center">
```
Comment (edi-odoo) [inline content/developer/howtos.rst:11]:
```suggestion
    <div class="row mb-4 center">
```

**Review by edi-odoo — CHANGES_REQUESTED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**